grapple/utils.py

```diff
@@ -125,6 +126,13 @@ def resolve_queryset(
 
     qs = qs.all() if id is None else qs.filter(pk=id)
 
+    # filter by in_menu
+    if in_menu is not None:
+        if in_menu:
+            qs = qs.in_menu()
+        else:
+            qs = qs.not_in_menu()
```

Comment:
this is the test failure

Reply:
It would be nice to get better output from ruff or better support for linting in vs code. but I have addressed these.